src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import React from 'react';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === activeTab.id ? 'is-active' : ''}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={(event) => {
+                  event.preventDefault();
+                  if (tab.id !== activeTabId) {
+                    onTabSelected(tab.id);
+                  }
+                }}
```

Comment:
You can try to bring out this function separately, to improve code readability.

Exanple:
```suggestion
                onClick={() => handleTabClick(tab.id)}
```